handler.go

```diff
@@ -20,6 +21,7 @@ type Response struct {
 // Request of function call
 type Request struct {
 	Body        []byte
+	Ctx         context.Context
```

Comment:
I have two questions about this one

1. should the name be `Context` so that it mimic the `http.Request` more closely? 
2. similarly, should it be a method instead of a property to even more mimic the `http.Request`?

what do you think @alexellis

Reply:
Playing catch-up here, I see it moved to encapsulated. If that's what stdlib does then I guess it's not a bad pattern to follow.